service/docs/source/runtime.rst

```diff
@@ -158,82 +145,63 @@ to profile unmodified applications, select and evaluate different GEOPM agent
+Agents in the compute nodes' tree hierarchy. The GEOPM Service package,
+which resides in the service directory of the GEOPM repository, provides
+the PlatformIO interface which abstracts reading signals and writing controls
+from the Agent within a compute node. This allows Agent implementations to
+be ported to various hardware platforms without modification.
+
+The libgeopm library can be called indirectly or directly within
+MPI applications, enabling application feedback to aid control
+decisions. Indirect calls are facilitated through GEOPM's integration with
+MPI and OpenMP via their profiling decorators. Direct calls are made through
+:doc:`geopm_prof(3)<geopm_prof.3>` or :doc:`geopm_fortran(3)<geopm_fortran.3>`
+interfaces. The application can be better integrated with the GEOPM runtime
+and controlled more accurately by marking up the compute application with
+profiling information obtained through these interfaces.
 
 Build Requirements
 ------------------
 
-When using the build system in the base of the GEOPM source repository
-to build the GEOPM Runtime some additional requirements must be
-met.  If the user is not interested in building the GEOPM Runtime,
-these extra build requirements may be ignored.  The user may also opt
-out of the specific GEOPM Runtime features enabled by any of these
-requirements by providing the appropriate disable flag to the base
-build configure command line.
-
-The GEOPM Runtime requires support for MPI, the Message Passing
-Interface, standard 2.2 or higher.  In many cases meeting this
-requirement will depend on the specific HPC resource being targeted
-based on documentation that is site specific.  The Intel MPI
-implementation may be used to meet this requirement.  The MPI
-requirement may also be met through HPC packaging systems like OpenHPC
-or Spack.  Additionally, the OpenMPI binaries are distributed with
-most major Linux distributions, and may also be used to satisfy this
-requirement.  This requirement can be met by installing the
-``openmpi-devel`` package version 1.7 or greater on RHEL and SLES
-Linux, and ``libopenmpi-dev`` on Ubuntu.
+When building the GEOPM Runtime from source, additional requirements must
+be met. Those uninterested in building the GEOPM Runtime can ignore these
+requirements, or by providing the disable flag to the configure command
+line, users may skip particular GEOPM Runtime features enabled by these
+requirements.
+
+The GEOPM Runtime requires MPI standards, Message Passing Interface,
+version 2.2 or later. Fulfilling this requirement generally depends on the
+specific HPC resource targeted based on site-specific documentation. The
+Intel MPI implementation, OpenHPC or Spack packaging systems, or OpenMPI
+binaries distributed with most major Linux distributions satisfy this
+requirement. For RHEL and SLES Linux, the requirement can be met by installing
+the ``openmpi-devel`` package version 1.7 or later, and ``libopenmpi-dev``
+on Ubuntu.
```

Comment:
"requires" -> "provides optional support for"

in general should reword this section.